src/controllers/User.controller.js

```diff
@@ -0,0 +1,6 @@
+const UserService = require('../services/User.service.js');
+const getAll = (req, res) => {
+  res.status(200).send(UserService.users);
```

Comment:
Same. I believe all your controller functions are the same, you should call the service first, and then send the status to the user, for example:

`const result = await UserService.users;`

`res.status(200).send(result)`